src/controllers/itemDetails/index.js

```diff
@@ -439,8 +439,20 @@ function renderName(item, container, context) {
     if (parentNameHtml.length) {
         if (parentNameLast) {
             // Music
+            // Determine if there are over 10 artists in the album. If so, concat with 'and X others.'
+            const artists = parentNameHtml[0].split(' / ');
+            const otherArtistCount = artists.length - 10;
+            let newParentNameHtml = artists.slice(0, 10).join(' / ');
+            newParentNameHtml = `${newParentNameHtml} and ${otherArtistCount} other artists.</br>`;
```

Comment:
I think we'd probably want to add an l10n string for this and then use `globalize.translate()` here. The l10n strings are defined under `src/strings`: https://github.com/jellyfin/jellyfin-web/blob/master/src/strings/en-us.json

It looks like the l10n lib supports string templates, so that should be possible with one string definition.

Reply:
Okay. I haven't worked with the l10n library before so I am going to look at other code on how to do it. I will submit my PR shortly. Let me know if it looks correct.